src/components/offer-card/offer-avatar-and-rating/OfferAvatarAndRating.jsx

```diff
@@ -0,0 +1,23 @@
+import Box from '@mui/material/Box'
+import Avatar from '@mui/material/Avatar'
+
+import AppRating from '~/components/app-rating/AppRating'
+
+import { styles } from '~/components/offer-card/offer-avatar-and-rating/OfferAvatarAndRating.styles'
+
+const OfferAvatarAndRating = ({ imgSrc, rating }) => {
+  return (
+    <Box sx={ styles.container }>
+      <Avatar src={ imgSrc } sx={ styles.avatar } />
+        <AppRating
+          sx={ styles.rating }
+          value={ rating }
+          precision={ 1 }
```

Comment:
i dont think we need precision by 1 on showing rating, only for adding new rating

Reply:
Done.